README.md

```diff
@@ -15,7 +15,7 @@
 ![screenshot](./screenshots/screen2.png)
 Covid Tracker is site used to check covid updates.
 ## [Live Demo](https://ncov-site.netlify.app/)
-## [Presentation](#)
```

Comment:
Please ensure that you stick to the design  [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size, and weight.
`Layout: composition and space between elements.`

Also, this is meant to be a mobile app.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/171836275-19b76feb-5044-49b4-a0ef-4d5620bd476d.png) | ![image](https://user-images.githubusercontent.com/10332499/171836485-7bdbe91c-aaf0-4f3b-8932-f0b8513c21ee.png)